CHANGELOG.md

```diff
@@ -0,0 +1 @@
+packages/gateway/CHANGELOG.md
```

Comment:
This doesn't seem working well;
https://github.com/graphql-hive/gateway/blob/docs/add-root-changelog/CHANGELOG.md

Reply:
Maybe it works probably it is a view bug in GH.